ctru-rs/src/error.rs

```diff
@@ -28,6 +29,7 @@ impl fmt::Debug for Error {
                 .field("summary", &R_SUMMARY(err))
                 .field("level", &R_LEVEL(err))
                 .finish(),
+            Error::ServiceAlreadyActive(service) => write!(f, "Service {service} already active"),
```

Comment:
We could use enum variants instead of the inner `&'static str` too if we wanted, since there is a finite well-known set of services. Might be annoying to keep in sync as we add new modules, though?